content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,299 @@
+life. Most contributors are not being paid to do this; we build it for fun, to solve a problem, to
+learn something, or for the simple love of the community. Many of us found our homes, our friends,
+and our careers here. The Release Team is honored to be a part of the continued growth of
+Kubernetes. 
+
+For the people who built it, for the people who release it, and for the furries who keep all of our
+clusters online, we present to you Kubernetes v1.30: Uwubernetes, the cutest release to date. The
+name is a portmanteau of “kubernetes” and “UwU,” an emoticon used to indicate happiness or cuteness.
+We’ve found joy here, but we’ve also brought joy from our outside lives that helps to make this
+community as weird and wonderful and welcoming as it is. We’re so happy to share our work with you. 
+
+UwU :heart:
+
+## Improvements that graduated to stable in Kubernetes v1.30 {#graduations-to-stable}
+
+_This is a selection of some of the improvements that are now stable following the v1.30 release._
+
+### Robust VolumeManager reconstruction after kubelet restart ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+This is a volume manager refactoring that allows the kubelet to populate additional information about how existing volumes are mounted during the kubelet startup. In general, this makes volume cleanup after kubelet restart or machine reboot more robust.
+
+This does not bring any changes for user or cluster administrators. We used the feature process and feature gate `NewVolumeManagerReconstruction` to be able to fall back to the previous behavior in case something goes wrong. Now that the feature is stable, the feature gate is locked and cannot be disabled.
+
+### Prevent unauthorized volume mode conversion during volume restore ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+For Kubernetes 1.30, the control plane always prevents unauthorized changes to volume modes when restoring
+a snapshot into a PersistentVolume. As a cluster administrator, you'll need to grant permissions to the appropriate identity principals (for example: ServiceAccounts representing a storage integration) if you
+need to allow that kind of change at restore time.
+
+For more information, see [converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode).
+
+### Pod Scheduling Readiness ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+_Pod scheduling readiness_ graduates to stable this release, after being promoted to beta in Kubernetes v1.27.
+
+This now-stable feature lets Kubernetes avoid trying to schedule a Pod that has been defined, when the cluster
+doesn't yet have the resources provisioned to allow actually binding that Pod to a node. That's not the only use
+case; the custom control on whether a Pod can be allowed to schedule also lets you implement quota mechanisms,
+security controls, and more.
```

Comment:
nit: do we want to add another line break here to match the rest of the formatting of the other sections?